src/Session/Session.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Pantheon\Terminus\Session;
+
+use Terminus\Caches\FileCache;
+use Terminus\Models\User;
+
+class Session {
+  /**
+   * @var FileCache
+   */
+  protected $cache;
+
+  /**
+   * @var object
+   */
+  protected $data;
+
+  /**
+   * Instantiates object, sets session data
+   */
+  public function __construct($fileCache) {
+    $this->cache = $fileCache;
+    $session     = $this->cache->getData('session');
+    $this->data  = $session;
+    if (empty($session)) {
+      $this->data = new \stdClass();
```

Comment:
If you feel like being lazy with your types, could I recommend `class SessionData extends \ArrayObject`?  You can use it like an array today, but it can also be used for typehinting, and you can add accessors to it later.  Win.